internal/querynodev2/segments/segment_loader.go

```diff
@@ -1504,20 +1531,20 @@ func getResourceUsageEstimateOfSegment(schema *schemapb.CollectionSchema, loadIn
 					loadInfo.GetSegmentID(),
 					fieldIndexInfo.GetBuildID())
 			}
+			segmentMemorySize += neededMemSize
 			if mmapEnabled {
```

Comment:
here is the most notable change here, I make it include segment memory usage when calculating for mmap segment, this is the point needed discussion

Reply:
Looks good to me. The memory cost on bloom filters should count.

My question is, provided that the memory cost on bloom filters are usually very low, why bother to change the code here? Is there any particular case that the BF memory cost made high?